tests/unit/query_plan.cpp

```diff
@@ -290,6 +290,31 @@ TYPED_TEST(TestPlanner, MatchMultiPattern) {
   DeleteListContent(&right_cartesian_ops);
 }
 
+TYPED_TEST(TestPlanner, MatchMultiPatternWithHashJoin) {
+  // Test MATCH (a:label)-[r1]->(b), (c:label)-[r2]->(d) WHERE c.id = a.id return a, b, c, d;
+  FakeDbAccessor dba;
+  const auto label_name = "label";
+  const auto property = PROPERTY_PAIR(dba, "id");
```

Comment:
also test with 2 different properties that need to match

Reply:
Added